### PR TITLE
Add nuget.org limits for metadata fields

### PR DESCRIPTION
**Pull request by joelverhagen**
Address https://github.com/NuGet/docs.microsoft.com-nuget/issues/2035

### CODE REVIEW

**Comment by anangaur** [comment]
Docs Build status updates of commit _[1ea1c4f](https://github.com/NuGet/docs.microsoft.com-nuget/commits/1ea1c4f55a47ec6c1a40450cd5a4ab8def5a3995)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2035/docs/reference/nuspec.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-2128) | [Details](#user-content-8224f5a0-0b348eb2)

<a id="8224f5a0-0b348eb2"></a>
### [docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2035/docs/reference/nuspec.md)
 - **Line 534, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 545, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 556, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 569, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 582, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 614, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 627, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 642, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 655, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 668, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 681, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 694, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 705, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 743, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 751, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 762, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008131744360106-2128%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=SJEhKbpmoUmQrI%2Fr1nMpSSB6z8m4b1Ik3XfaQ7%2BKjAE%3D&st=2020-08-13T17%3A40%3A16Z&se=2020-09-13T17%3A45%3A16Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by anangaur** [comment]
Docs Build status updates of commit _[8e40dd4](https://github.com/NuGet/docs.microsoft.com-nuget/commits/8e40dd46abb24f54fca56f2c4d0bdcedaa37d1c0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2035/docs/reference/nuspec.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-2128) | [Details](#user-content-8224f5a0-487c1d37)

<a id="8224f5a0-487c1d37"></a>
### [docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2035/docs/reference/nuspec.md)
 - **Line 520, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 531, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 542, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 555, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 568, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 600, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 613, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 628, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 641, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 654, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 667, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 680, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 691, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 729, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 737, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 748, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008131746336150-2128%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=MK%2BtV5wgj7i70vj69i7kpoD9GRK%2FIoAitGBIXeWAuP0%3D&st=2020-08-13T17%3A42%3A07Z&se=2020-09-13T17%3A47%3A07Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by anangaur** [comment]
Docs Build status updates of commit _[c7a0bbb](https://github.com/NuGet/docs.microsoft.com-nuget/commits/c7a0bbb4e28c0f704cac42c5c9675383a1dee043)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2035/docs/reference/nuspec.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-2128) | [Details](#user-content-8224f5a0-1dd75640)

<a id="8224f5a0-1dd75640"></a>
### [docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2035/docs/reference/nuspec.md)
 - **Line 520, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 531, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 542, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 555, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 568, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 600, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 613, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 628, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 641, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 654, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 667, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 680, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 691, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 729, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 737, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 748, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008131748572928-2128%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=7Mn1ZT6ExbiVsFhgctmwMlZS83gBBzcgUyoyEnOfDX0%3D&st=2020-08-13T17%3A44%3A27Z&se=2020-09-13T17%3A49%3A27Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/reference/nuspec.md [hunk 1]
```diff
@@ -215,9 +240,15 @@ For example:
 </package>
 ```
 
+When uploading a package to nuget.org, the `type` attribute is limited to 100 characters and the `url` attribute is
+> limited to 4000 characters.
```
Comment (loic-sharma) [inline docs/reference/nuspec.md:244]: It looks like the `>` here was added by accident. Same comment for line 250 and 351.

Comment (loic-sharma) [reply docs/reference/nuspec.md:244]: See:

* https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-2128#repository
* https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-2128#title
* https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-2128#dependencies-element

**Comment by anangaur** [comment]
Docs Build status updates of commit _[bb6b374](https://github.com/NuGet/docs.microsoft.com-nuget/commits/bb6b3744d3a55a114f48c8577b2bd235f901c224)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2035/docs/reference/nuspec.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-2128) | [Details](#user-content-8224f5a0-97c64e09)

<a id="8224f5a0-97c64e09"></a>
### [docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-2035/docs/reference/nuspec.md)
 - **Line 529, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 540, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 551, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 564, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 577, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 609, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 622, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 637, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 650, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 663, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 676, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 689, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 700, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 738, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 746, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 757, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C17%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008171632336203-2128%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=hFl0bz9nF1LBZiGAtxbw3FuE2ToszvnRmd9t3CDxCY8%3D&st=2020-08-17T16%3A28%3A06Z&se=2020-09-17T16%3A33%3A06Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**